app/src/main/assets/OpenAPSSMBDynamicISF/determine-basal.js

```diff
@@ -501,9 +510,41 @@ var determine_basal = function determine_basal(glucose_status, currenttemp, iob_
         return rT;
     }
 
-    // min_bg of 90 -> threshold of 65, 100 -> 70 110 -> 75, and 130 -> 85
-    var threshold = min_bg - 0.5*(min_bg-40);
+    // min_bg of 90 -> threshold of 65, 100 -> 70 110 -> 75, and 130 -> 85, or if specified by user, take that value
+        var lgsThreshold = profile.lgsThreshold;
+        console.error("Profile LGS Threshold set to"+lgsThreshold+"; ");
+        //if(lgsThreshold < 5.6)
+        if(lgsThreshold > 3.2 && lgsThreshold < 5.6){
+            lgsThreshold = (18 * lgsThreshold);
+            lgsThreshold = round(lgsThreshold, 2);
+            console.error("LGS Threshold converted to"+lgsThreshold+"; ");
+        }else{
+            lgsThreshold;
+        }
+        var threshold = min_bg - 0.5*(min_bg-40);
+        var oldThreshold = threshold;
+        if(lgsThreshold < 65 || lgsThreshold > 120) {
+            threshold;
+            }
+        else if( lgsThreshold < threshold){
+            threshold;
```

Comment:
It does nothing

Reply:
Yes. If the user LGS threshold is outside the limits that we allow, including the calculated threshold, we leave the calculated threshold. It's only when user defined value is higher than cacluated value, that we replace the calculated value.